schemas/extractors.schema.json

```diff
@@ -29,6 +29,8 @@
                 "maintenance_status": {},
                 "domain_url": {},
                 "keywords": {},
+                "usage": {},
```

Comment:
@tayloramurphy are these keys expected to also be returned by the API? If not we can add them to https://github.com/meltano/hub/blob/40ae36b69c691c065aacafe75b10252d22ef2f64/_plugins/meltano_json_generator.rb#L68 to exclude them from the generated static files. Also this makes me wonder if we should split the schemas between plugin and API in a follow on because theyre starting to diverge with some keys excluded. The schemas were orignally created to valid the API static files but overlapped with the plugin files also, but since we have a few keys were dropping it no longer validates perfectly i.e. validation would pass even though we dont allow this key in the API.

Reply:
Also if these are used for validating the plugin definition the maybe we should add them to the others as well because they might need custom usage notes too.

Reply:
@pnadolny13 let's leave them in for now since they are relevant instructions similar to the settings instructions.

I plan on adding them to the other plugins in #550